### PR TITLE
added divider

### PR DESCRIPTION
**Pull request by wmosienski**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "no-empty": 2,
         "no-extra-boolean-cast": 1,
         "linebreak-style": [0],
+        "object-curly-newline": 0,
```
Comment (solleOlle) [inline .eslintrc.json:30]: why don't you create a separate PR for settings?

File: src/stories/core/Divider/Divider.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
+                {context.args.vertical ? (
+                    <div style={{ display: 'flex', flexDirection: 'row' }}>
+                        <div style={{ padding: '40px', backgroundColor: 'grey' }}>element</div>
+                        <Story />
+                        <div style={{ padding: '40px', backgroundColor: 'grey' }}>element</div>
+                    </div>
+                ) : (
+                    <div style={{ display: 'flex', flexDirection: 'column' }}>
+                        <div style={{ padding: '40px', backgroundColor: 'grey' }}>element</div>
```
Comment (solleOlle) [inline src/stories/core/Divider/Divider.stories.jsx:18]: you repeat 
<div style={{ display: 'flex', flexDirection: 'row' }}>
    <div style={{ padding: '40px', backgroundColor: 'grey' }}>element</div>
    <Story />
    <div style={{ padding: '40px', backgroundColor: 'grey' }}>element</div>
</div>

for changing direction only
could you create a variable for that? and I don't really like inline styles... could we do something with it?

Comment (wmosienski) [reply src/stories/core/Divider/Divider.stories.jsx:18]: I'm on it :)

Comment (wmosienski) [reply src/stories/core/Divider/Divider.stories.jsx]: @solleOlle done

**Pull request merged**